### ISSUE TITLE
Pie Chart and Bar chart are no longer displaying.

### ISSUE THREAD

**Issue by archanakeshireddy** [google/google-visualization-issues#2989]
We are using GWT 2.8.2 and GWT Visualization 1.1.1. 

Pie Chart in our application is no longer displaying.
A red box with message 'this.rd is null' is displaying.

Bar chart is also no longer displaying.

It looks like gwt-visualization code is internally downloading http://www.google.com/jsapi?callback=__gwt_AjaxLoader_onLoad
and this is internally calling https://www.gstatic.com/charts/52/loader.js

<img width="148" alt="Screen Shot 2023-03-01 at 4 35 18 PM" src="https://user-images.githubusercontent.com/66965375/222271445-ccf723e0-4d4e-439b-ab93-759e066bb753.png">

**Comment by dlaliberte** [comment]
Yes, the old jsapi loader redirects internally to the new gstatiic.com/charts/loader.js.  Since you are getting the v52 loader, your google.load call is probably loading "1.1" which maps to the new 'upcoming' version.  

GWT has long been deprecated, and any lingering support is gradually falling apart, and will eventually fail completely.    So your best bet is to figure out how to update your code to use the latest version.  See https://developers.google.com/chart/interactive/docs/basic_load_libs for details.

**Comment by archanakeshireddy** [comment]
Thanks. I looked into the gwt visualization code and it has below code after loading loader.js it is calling below method.
do you know if we change the visualization code to load https://www.gstatic.com/charts/51/loader.js will fix this issue.
    $wnd.google.load(api, version, settings);

**Comment by dlaliberte** [comment]
It is possible that v51 will still work with GWT.  Worth a try.  You should use '51' as the version, if you want it to stick with that version for long term.

**Comment by mike-targetr** [comment]
We also encountered this problem today. I can confirm that loading version 51 resolved the issue:

`VisualizationUtils.loadVisualizationApi("51", onLoad, "corechart");`

**Comment by dlaliberte** [comment]
Thanks mike-targetr.  Since GWT was deprecated long ago, and v52 breaks the support of GWT for some unknown reason, it is reasonable to require anyone who still needs GWT support to use v51 instead.
